scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1909,6 +1909,39 @@ class FormatOps(
     }
   }
 
+  object OptionalBraces {
+    def unapply(ft: FormatToken): Option[(FormatToken, Tree)] = ft.left match {
+      case _ if !dialect.allowSignificantIndentation => None
+      case _: T.Comment | _: T.LeftBrace => None
+      case _: T.Colon => // Optional braces in templates after `:`
+        ft.meta.leftOwner match {
+          case t: Template =>
+            val ok = t.tokens.headOption.contains(ft.left)
```

Comment:
this uses the fact that template starts with `:`, although i would have expected colon to serve as a separator.

Reply:
this doesn't work either. if there are extends, colon is not the first token. also switched to `templateCurly`.